scheduler/src/cook/mesos/api.clj

```diff
@@ -935,6 +946,23 @@
       (log/debug e "Unable to retrieve directory path for" task-id "on agent" agent-hostname)
       nil)))
 
+(defn compute-cluster-entity->map
+  [entity]
+  (cond-> {:compute-cluster-name (:compute-cluster/cluster-name entity)}
+    (= :compute-cluster.type/mesos (:compute-cluster/type entity))
+          (assoc :mesos {:framework-id (:compute-cluster/mesos-framework-id entity)})))
```

Comment:
Indentation seems off here

Reply:
Yeah, my cursive wants to mis-indent cond->